README.md

```diff
@@ -9,6 +9,19 @@ conda create -n mmda python=3.8
 pip install -e '.[dev,<extras_require section from setup.py>]'
 ```
 
+## Unit testing
+```bash
+pytest ./tests
+```
+for latest failed test
+```bash
+pytest ./tests --lf
+```
+for specific test name of class name
+```bash
+pytest ./tests -k <'name_of_class_or_unit_test'>
```

Comment:
Maybe an example of how a working command would look exactly?

Reply:
Sure thing, I've also added pytest.ini and packages for runing tests on multiple cpus.